docs/modules/ROOT/pages/management-center-ldap.adoc

```diff
@@ -0,0 +1,73 @@
+= LDAP Security Provider
+:description: You can use your existing LDAP server for authentication/authorization on the Management Center.
+
+{description}
+
+See xref:management-center:deploy-manage:ldap.adoc[Management Center Documentation] for more detailed information.
+
+== Setting Up the LDAP Security Provider
+
+To set up LDAP, you need to configure the `securityProvider.ldap` section in the Management Center CR.
+The following are the required field to configure the LDAP Security provider.
+
+[cols="30%m,70%a"]
+|===
+|Field|Description
+
+|`credentialsSecretName`
+| The name of the secret that contains `username` and `password` keys of a user that has admin privileges on the LDAP server. The `username` must be the DN of the user. It is used to connect to the server when authenticating users.
+
```

Comment:
Maybe it is better to show the expanded word for DN when it is used for the first time. Something like:

"... must be the distinguished name (DN) of the user. ..."